examples/ERP/plot_classify_EEG_quantum.py

```diff
@@ -0,0 +1,127 @@
+"""
+====================================================================
+ERP EEG decoding with Quantum Classifier.
+====================================================================
+
+Decoding applied to EEG data in sensor space decomposed using Xdawn.
+After spatial filtering, covariances matrices are estimated, then projected in
+the tangent space and classified with a quantum classifier
+
+"""
+# Authors: Gregoire Cattan, Alexandre Barachant <alexandre.barachant@gmail.com>
+# Modified from plot_classify_EEG_tangentspace.py
+# License: BSD (3-clause)
+
+import numpy as np
+
+from pyriemann.estimation import XdawnCovariances
+from pyriemann.tangentspace import TangentSpace
+from pyriemann.classification import QuanticSVM
+
+import mne
+from mne import io
+from mne.datasets import sample
+
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import confusion_matrix, ConfusionMatrixDisplay
+
+from matplotlib import pyplot as plt
+
+# cvxpy is not correctly imported due to wheel not building
+# in the doc pipeline
+__cvxpy__ = True
+try:
+    import cvxpy
```

Comment:
This hack is here just for the documentation pipeline, in order for the doc to build without cvxpy dependency.